src/map/pc.cpp

```diff
@@ -4860,6 +4861,10 @@ char pc_additem(struct map_session_data *sd,struct item *item,int amount,e_log_p
 		if( item->equipSwitch )
 			sd->inventory.u.items_inventory[i].equipSwitch = 0;
 
+		for (j = 0; j < MAX_FAVORITES; j++) {
+			if (sd->status.favs[j] == item->nameid) sd->inventory.u.items_inventory[i].favorite = 1;
```

Comment:
absolutely no idea what you want here, no else branch. To be honest formatting is of zero importance to me so it might be more effective if you submitted the change you want like in the first two cases so I can just click the commit button and have it done.

Reply:
Yes, I meant formatting, sorry for not being clear. It makes it hard to read when everything is bunched together on a single line. Remember, white space is free!

Reply:
added a line break. Personally I prefer if there is a single instruction in an if statement to be in the same line - makes sure you don't accidentally read the second line without noticing the condition part.